src/jobs/terraform_deploy.yml

```diff
@@ -48,4 +54,12 @@ steps:
   # Real work
   - run: terraform workspace new <<parameters.tf-workspace>>
   - run: terraform init -upgrade <<parameters.main-dir>>
-  - run: terraform apply -input=false -auto-approve <<parameters.main-dir>>
+  - when:
```

Comment:
instead of needing all this to use a json file why not use tfvars. Then TF will automatically suck it in if one exists instead of having to modify the pipeline to specify the parameter. Just thinking of easy path with less things for someone to forget.

Reply:
@Stromweld - Per our chat, Terraform will pull in the JSON file automatically. I believe the issue here is that TF is being run from another directory, and does not automatically detect the file. We could tests this a bit more.

Details here: https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files